test/unit/concerns/host_managed_extensions_test.rb

```diff
@@ -0,0 +1,35 @@
+require 'test_plugin_helper'
+
+class HostManagedExtensionsTest < ActiveSupport::TestCase
+  before do
+    @role1 = FactoryGirl.create(:ansible_role)
+    @role2 = FactoryGirl.create(:ansible_role)
+    @role3 = FactoryGirl.create(:ansible_role)
```

Comment:
Not directly but I wanted to have one more role there to test that I'm actually not displaying what I shouldn't.